app/src/main/res/values/strings.xml

```diff
@@ -132,11 +132,16 @@
 
     <string name="dashboard_risky_encounter_title">Za posledních 14 dní žádné rizikové setkání</string>
     <string name="dashboard_risky_encounter_subtitle">Poslední aktualizace 14. 5. 2020 v 17:30.\nAktualizace probíhá jednou za 24 hodin</string>
+    <plurals name="dashboard_risky_encounter_title_bad">
+        <item quantity="zero">Za posledních 14 dní %1$d rizikových setkání</item>
```

Comment:
The number of days should be configurable via RC. Maybe we will need to change it in the future.

Reply:
did not do RX config and translations yet as from what I understood the texts might not be final yet